checker/src/main/java/org/checkerframework/checker/resourceleak/ResourceLeakTransfer.java

```diff
@@ -83,6 +85,20 @@ public TransferResult<CFValue, CFStore> visitMethodInvocation(
       accumulate(accumulationTarget, result, methodName);
     }
 
+    // If the invoked method has @Owning parameters, treat them as if their must-call
+    // methods have all been called.
+    ExecutableElement methodElt = node.getTarget().getMethod();
+    // Have to use an indexed for-loop, because the loop accesses both the variable element
+    // (to check for @Owning) and the corresponding Node (to call accumulate()).
+    for (int i = 0; i < methodElt.getParameters().size(); i++) {
+      VariableElement param = methodElt.getParameters().get(i);
+      if (rlTypeFactory.hasOwning(param)) {
+        List<String> mcMethods = rlTypeFactory.getMustCallValue(param);
+        Node correspondingArgument = node.getArgument(i);
+        accumulate(correspondingArgument, result, mcMethods.toArray(new String[0]));
```

Comment:
Sorry I don't fully remember, this will add `mcMethods` to the called methods for both the normal and exceptional successor of this call, correct?

Reply:
Nevermind, answered my own question, it will work for exceptional successors also, via this line https://github.com/typetools/checker-framework/blob/e3a6e65bd65eeb92f4063f975a151d109824b063/checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsTransfer.java#L116-L116